Sources/NonEmpty/NonEmpty.swift

```diff
@@ -1,21 +1,152 @@
+@dynamicMemberLookup
```

Comment:
Dynamic member lookup seems useful for this kind of thing.

Reply:
This is a smart use of it, saves you having to forward all the computed properties manually.